### DISCUSSION TITLE
[Feature] edit pictures online

### DISCUSSION THREAD

**Discussion by Christopher-Songs** [immich-app/immich#13374] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I hope this message finds you well. I am writing to inquire whether your platform supports basic online editing features, such as rotating or cropping, without the need to download files locally beforehand.

Thank you for your assistance.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1991
